### PR TITLE
feat: add deepin-system-monitor log adaptation work

### PR DESCRIPTION
**Pull request by add-uos**
add deepin-system-monitor log adaptation work

Log: add deepin-system-monitor log adaptation work

Task: https://pms.uniontech.com/task-view-341559.html

### CODE REVIEW

**Review by pengfeixx — COMMENTED** [review]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

File: deepin-system-monitor-daemon/src/logger.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+#include <QObject>
+#include <dtkcore_global.h>
+
+DCORE_BEGIN_NAMESPACE
+class DConfig;
+DCORE_END_NAMESPACE
+
+class MLogger : public QObject
```
Comment (rb-union) [inline deepin-system-monitor-daemon/src/logger.h:12]: 每个工程都有相同的类，功能逻辑没有太大插件，初始参数不同，考虑下抽为复用的utils类？

**Review by max-lvs — APPROVED** [review]

**Review by pengfeixx — APPROVED** [review]

**Pull request merged**